drivers/applications/driver/driver.go

```diff
@@ -0,0 +1,84 @@
+package driver
+
+import (
+	"context"
+
+	. "github.com/portworx/torpedo/drivers/applications/apptypes"
+	. "github.com/portworx/torpedo/drivers/applications/mysql"
+	. "github.com/portworx/torpedo/drivers/applications/postgres"
+	. "github.com/portworx/torpedo/drivers/utilities"
+)
+
+type ApplicationDriver interface {
+	DefaultPort() int
+
+	DefaultDBName() string
+
+	ExecuteCommand(commands []string, ctx context.Context) error
+
+	StartData(command <-chan string, ctx context.Context) error
+
+	CheckDataPresent(selectQueries []string, ctx context.Context) error
+
+	UpdateDataCommands(count int, identifier string)
+
+	InsertBackupData(ctx context.Context, identifier string, commnads []string) error
+
+	GetBackupData(identifier string) []string
+
+	GetRandomDataCommands(count int) map[string][]string
+
+	AddDataCommands(identifier string, commands map[string][]string)
+
+	GetApplicationType() string
+
+	GetNamespace() string
+}
```

Comment:
Add doc string for each of these functions

Reply:
Added doc strings